packages/rx_bloc_cli/example/README.md

```diff
@@ -353,6 +353,8 @@ Due to the way fonts are loaded in tests, any custom fonts you intend to golden
 
 In order for the goldens to be generated, we have provided VS Code and IDEA run configurations, as well as an executable `bin/generate_goldens.sh`. The golden masters will be located in `goldens/light_theme` and `goldens/dark_theme`. The `failures` folder is used in case of any mismatched tests.
 
+Using smart widgets with dependency injection might cause golden tests to fail. We developed a solution to remove dependencies during test mode, which gives you the opportunity to mock them. Read the documentation [here][golden_tests_for_smart_widgets_link].
```

Comment:
I believe this link makes the link the main README.md file redundant

Reply:
@StanevPrime Should we remove this link?

Reply:
Removed link in the rx_bloc_cli package readme.